test/src/main/java/org/axonframework/test/aggregate/Reporter.java

```diff
@@ -225,6 +223,41 @@ public void reportDifferentEventContents(Class<?> eventType, Field field, Object
         throw new AxonAssertionError(sb.toString());
     }
 
+    /**
+     * @param eventType         The (runtime) type of event the difference was found in
+     * @param missingEntries    The expected key-value pairs that where not present in the metadata
+     * @param additionalEntries Key-value pairs that where present in the metadata but not expected
+     */
+    public void reportDifferentMetaData(Class<?> eventType, Map<String, Object> missingEntries, Map<String, Object> additionalEntries) {
+        StringBuilder sb = new StringBuilder("One of the events contained different metadata than expected");
+        sb.append(NEWLINE)
+                .append(NEWLINE)
+                .append("In an event of type [")
+                .append(eventType.getSimpleName())
+                .append("], ");
+        if (!additionalEntries.isEmpty()) {
+            sb.append("metadata entries" + NEWLINE)
+                    .append("[");
+            for (Map.Entry<String, Object> entry : additionalEntries.entrySet()) {
+                sb.append(entryAsString(entry) + ",");
```

Comment:
Nit: wouldn't changing the separator to `", "` give a clearer String of unexpected additional metadata entries?